app/noScript.js

```diff
@@ -77,7 +81,7 @@ function onHeadersReceived (details) {
   }
 
   let csp = details.responseHeaders[cspHeaderName] || []
-  csp.push("script-src 'none'")
+  csp.push('script-src chrome-extension://*')
```

Comment:
I have some changes for that in a PR I'm working on. I'll verify that everything works without chrome-extension:// and if not, I'll check the noscript ui.